c2corg_ui/static/js/documentediting.js

```diff
@@ -185,21 +186,20 @@ app.DocumentEditingController = function($scope, $element, $attrs,
    */
   this.api_ = appApi;
 
-  // allow association only new outing  to existing route
+  // allow association only new outing to existing route
   if (this.ngeoLocation_.hasParam('routes')) {
     this.urlParams_ = {'routes': this.ngeoLocation_.getParam('routes')};
     this.pushDocToAssociations_();
   }
 
-  // When creating a new document, the model is not created until
-  // the form is touched. At least create an empty object.
-  this.scope_[this.modelName_] = this.scope_['document'] = this.documentService.document;
```

Comment:
In the final page it is embedded in the HTML. Which means it is downloaded every time the page is downloaded. I just wondered if there was a way to download it separately like an image. But perhaps it wouldn't give much gain?